osquery/distributed/distributed.cpp

```diff
@@ -183,6 +166,21 @@ Status Distributed::acceptWork(const std::string& work) {
               1,
               "Distributed discovery query does not have complete attributes");
         }
+        if (doesQueryRequireSignature(query)) {
+          if (signatures.count(node.first + "_disc") > 0) {
```

Comment:
Disc is here because to not protect discovery queries is dangerous. Discovery queries use the _disc affix already so changing it would be changes in other places + an API Change. But I agree with your comment on _disc, I should have just used the whole word.